src/controllers/auth.controller.ts

```diff
@@ -49,6 +50,17 @@ class AuthController extends BaseController {
       this.errorHandler(res, error);
     }
   }
+  async profile(req: Request, res: Response) {
+    try {
+      const token = req.headers.authorization?.split(" ")[1];
```

Comment:
This is bad implementation, we just use auth middleware

Reply:
implementantion corrected in the new commit